src/replication.c

```diff
@@ -593,6 +593,8 @@ void replicationFeedMonitors(client *c, list *monitors, int dictid, robj **argv,
     while((ln = listNext(&li))) {
         client *monitor = ln->value;
         addReply(monitor,cmdobj);
+        /* Note: If someday we wanna update the monitors' memory usage accurately
+         * that we need to re-add an `updateClientMemoryUsage()` call here. */
```

Comment:
i think it's better to put the comment in updateClientMemUsageAndBucket and mention replicationFeedMonitors there (no one will come looking for it here, but if we modify the code there, we need to come here)

Reply:
Done